app/templates/components/profile-nav.hbs

```diff
@@ -81,6 +81,12 @@
             {{#link-to 'organization.repositories' model }}Repositories{{/link-to}}
           </li>
 
+          {{#if showOrganizationSettings}}
+            <li data-test-org-settings-tab>
+              {{#link-to 'organization.settings'}}Settings{{/link-to}}
```

Comment:
Doesn't it require `organization` model to be provided, similar to the Repositories link above? I mean, there may be multiple organizations in the list, and usually it's mandatory to pass a model to resolve the route. Otherwise, how would it know, for which org to show the settings?

Reply:
Huh yeah I think you're right. It's weird that it went unnoticed, but maybe it's because I'm only the admin/owner of 1 organization. Either way, I'll update this to pass in `model`

Reply:
Change made 👍